config.toml

```diff
@@ -7,7 +7,7 @@ enableRobotsTXT = true
 
 disableKinds = ["taxonomy", "taxonomyTerm"]
 
-ignoreFiles = [ "^OWNERS$", "README.md", "^node_modules$" ]
+ignoreFiles = [ "^OWNERS$", "README.md", "^node_modules$", "content/en/docs/doc-contributor-tools" ]
```

Comment:
Is there a compelling reason to add the new folder to `/content/en/docs/` instead of the repo top level?

Reply:
They are docs-specific and probably wouldn't be used by people working on the blog? I dunno, just wanted to keep it clean.